include/flatbuffers/code_generators.h

```diff
@@ -29,7 +29,7 @@ class BaseGenerator {
     EnsureDirExists(path.c_str());
     if (parser.opts.one_file) return path;
     std::string namespace_dir = path;  // Either empty or ends in separator.
-    auto &namespaces = ns.components;
+    auto namespaces = PrefixedNamespaceComponents(ns, parser.opts.lang);
```

Comment:
are there any other places where ns,components is used that this needs to be called instead?